dev/devicelab/bin/tasks/new_gallery_chrome_run_test.dart

```diff
@@ -0,0 +1,97 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:shelf/shelf.dart';
+import 'package:shelf/shelf_io.dart' as shelf_io;
+import 'package:shelf_static/shelf_static.dart';
+
+import 'package:flutter_devicelab/framework/browser.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+
+/// The port to serve the compiled New Flutter Gallery app.
+const int serverPort = 9999;
+
+Future<void> main() async {
+  await task(const NewGalleryChromeRunTest().run);
+}
+
+/// Checks that the New Flutter Gallery runs successfully on Chrome.
+class NewGalleryChromeRunTest {
+  const NewGalleryChromeRunTest();
+
+  /// Runs the test.
+  Future<TaskResult> run() async {
+    await gitClone(path: 'temp', repo: 'https://github.com/flutter/gallery.git');
+
+    await inDirectory<TaskResult>('temp/gallery', () async {
+      await flutter('doctor');
+      await flutter('packages', options: <String>['get']);
+
+      await evalFlutter('build', options: <String>[
+        'web',
+        '-v',
+        '--release',
+        '--no-pub',
+      ], environment: <String, String>{
+        'FLUTTER_WEB': 'true',
+      });
+
+      HttpServer server;
+      final Cascade cascade = Cascade().add(createStaticHandler('temp/gallery/build/web'));
+      server = await HttpServer.bind('localhost', serverPort);
+
+      Chrome chrome;
+
+      shelf_io.serveRequests(server, cascade.handler);
+
+      final ChromeOptions options = ChromeOptions(
+        url: 'http://localhost:$serverPort/index.html',
+        windowHeight: 1024,
+        windowWidth: 1024,
+        headless: false,
+      );
+
+      chrome = await Chrome.launch(
+        options,
+        onError: (dynamic error) {print('error $error happened');},
+        workingDirectory: cwd,
+      );
+
+      await Future<String>.delayed(const Duration(seconds: 5), () => 'done');
```

Comment:
How do we know that this is successful?

Reply:
I think if there are any errors, it should raise an exception, which would fail the test.
I'll try to rewrite it with the method in `web_incremental_test.dart`.

Reply:
I rewrote this test.
Now, if any exceptions were raised by the Gallery in the first 5 seconds, the test will fail.